src/sync/queries.ts

```diff
@@ -52,7 +52,7 @@ export const getCommits = `query ($owner: String!, $repo: String!, $per_page: In
                   message
                   oid
                   url
-                  changedFiles
+                  ${includeChangedFiles ? "changedFiles" : ""}
```

Comment:
Has this been tested? Wouldn't graphql shit itself if it tried to query target."" ?

Reply:
GraphQL doesn't care about empty lines. Tested with the GraphQL explorer and with a local sync job